tests/integration/test_tenants.py

```diff
@@ -27,12 +27,12 @@ def test_tenants_POST(client, auth_headers):
         "lastName": "The Dog",
         "phone": "111-111-1111",
         "propertyID": 1,
-        "staffIDs": [1, 2],
-        "unitNum": "237"
+        "staffIDs": [1, 2]
     }
 
     response = client.post(endpoint, json=newTenant,
                            headers=auth_headers["admin"])
+    print(response)
```

Comment:
Let's remove this print statement.

Reply:
son of a... thought I took that out last night. just pushed update